src/common/display_screen.h

```diff
@@ -17,6 +17,8 @@
 #ifndef DISPLAY_SCREEN_H
 #define DISPLAY_SCREEN_H
 
+#include <algorithm>
```

Comment:
It seems Windows defines `min` as `__min` in `stdlib.h`. Instead of using ifdef guards on the platform just use `min` from `algorithm.h`. We already link against `algorithm`.